src/components/event-listing/EventListing.tsx

```diff
@@ -60,7 +96,17 @@ const EventListing: React.FC<EventProps> = ({
           <Text strong>{dateFormat(details.start, 'shortTime', true)}</Text>
           <ThinDivider />
           <Paragraph ellipsis={{ rows: 5 }}>{details.description}</Paragraph>
-          <GreenButton to={`/events/${id}`}>Register</GreenButton>
+          {admin ? (
+            <AdminButtonWrapper>
```

Comment:
🔧 Currently this leaves admin with no way to access the event, for now can we keep the "Register" button on admin too? I know this conflicts with the design but we dont want to cut off navigation